aws/lambda/pytorch-auto-revert/pytorch_auto_revert/testers/autorevert_v2.py

```diff
@@ -70,9 +72,9 @@ def autorevert_v2(
     logging.info("[v2] Candidate action groups: %d", len(groups))
 
     # Support toggling specific kinds of actions via flags
-    if not do_revert:
+    if revert_action == RestartRevertAction.IGNORE:
```

Comment:
it doesn't make sense to duplicate this check here and inside `execute_restart` / `execute_revert` methods